run-poc/redefine-mst-parser.scm

```diff
@@ -230,17 +230,16 @@
 		(lambda (dj) (if (not (is-oversize? dj)) (count-one-atom dj)))
 		(make-sections parse)
 	)
-	(if EXPORT-MST
+	(if (not (equal? EXPORT-MST "#f"))
```

Comment:
OH, I see the problem.  This one change is forcing you to use `"#f"` I think the original form was correct:  either `EXPORT-MST` is false, or `EXPORT-MST` is a string holding a filename.  Maybe you should keep the old version, and just change `EXPORT-MST` to `MST-FILENAME`

Reply:
The problem with the original approach is that I need to pass through netcat a variable that is either a string or a bool, so I need to pass it using 
> 			print NC "($ARGV[2] \"$buffer\" \"$ARGV[3]\" '(@dist_mult) \"$ARGV[5]\")\n";

I think I will use your suggestion of using "none" as a keyword for no exporting.